Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SectionedCollectionViewController.swift
+//  SectionedCollectionView
+//
+//  Created by Esteban Arrua on 6/7/18.
+//  Copyright © 2018 Hattrick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import SectionedCollectionView
```

Comment:
is this import needed? shouldn't it be included with the Rx version?

Reply:
Yes, it is needed. The Rx only include de rx function, to define and use the SectionedCollectionView function and clases it needed import the SectionedCollectionView. I search in RxAlamonfire and RxPermission, and they did the same.

Reply:
cool